src/lib/journey/stages/generic.svelte

```diff
@@ -68,12 +69,13 @@
     }
   });
 
+  onMount(() => captureLinks(linkWrapper, journey));
+
   $: {
-    console.log(formNeedsFocus);
     checkValidation = initCheckValidation();
     callbackMetadataArray = buildCallbackMetadata(step, checkValidation);
     stepMetadata = buildStepMetadata(callbackMetadataArray);
-    failureMessageKey = convertStringToKey(failureMessage);
+    formMessageKey = convertStringToKey(form?.message);
```

Comment:
I think we should move this into the `journey.store` to clean up the surface area for the stage component even more. I _really_ wanted to do it, but I restrained myself.